spec/features/start_spec.rb

```diff
@@ -0,0 +1,71 @@
+# feature "Starting the game" do
+#   scenario "Start the first game" do
+#     register_and_play
+#     click_link 'START'
+#     expect(page).to have_content "Dave, choose one of the options:"
+#   end
+# end
+
+
+feature "playing a game" do
+  SEED = 221563
+
+  before do
+    register_and_play
+  end
+
+  scenario "options presentation" do
+    expect(page).to have_content "Rock"
+    expect(page).to have_content "Paper"
+    expect(page).to have_content "Scissors"
```

Comment:
Not sure if the one expect per block still applies to Capybara, but if it does you could split these up.